git-service/src/softserve/actions.rs

```diff
@@ -0,0 +1,53 @@
+use crate::softserve::config;
+use crate::softserve::models::{CreateRepoResponse, CreateTokenResponse, CreateUserResponse};
+use crate::softserve::ssh::execute_command;
+
+pub fn create_user(username: &str) -> Result<CreateUserResponse, Box<dyn std::error::Error>> {
+    let command = format!("user create {}", username);
+    Ok(execute_command(&command).map(|_| CreateUserResponse {
+        username: username.to_string(),
+        message: format!("User {} created successfully on Soft Serve", username),
+    })?)
+}
+
+pub fn create_token(token_name: &str) -> Result<CreateTokenResponse, Box<dyn std::error::Error>> {
+    let command = format!("token create {}", token_name);
+    let output = execute_command(&command)?;
+    let token = output.trim().to_string();
+    Ok(CreateTokenResponse {
+        token_name: token_name.to_string(),
+        token,
+    })
+}
+
+pub fn create_repo(
+    repo_name: &str,
+    repo_url: &str,
+) -> Result<CreateRepoResponse, Box<dyn std::error::Error>> {
+    let import_command = format!("repo import {} {}", repo_name, repo_url);
+    execute_command(&import_command)?;
+
+    let token_command = format!("token create {}_token", repo_name);
+    let token_output = execute_command(&token_command)?;
+    let token = token_output.trim().to_string();
+
+    let server_url = self::config::get_server_url();
+    let scheme = if self::config::get_is_prod() { "https" } else { "http" };
```

Comment:
```suggestion
    let server_url = config::get_server_url();
    let scheme = if config::get_is_prod() { "https" } else { "http" };
```

`self` isn't necessary here.

Reply:
I used this base on your previous review, which of makes sense, I know there are couple of ways to ref import. Is it because I now moved it to another file from mod? Kindly share some thought here. 

 ref: [Here](https://github.com/Extheoisah/hxckr-infra/pull/4#discussion_r1741679600)

Reply:
Yeah the previous feedback was left for `mod.rs`.  In that file you are declaring the config module.  However in this file you are using the crate directly so it isn't available locally (unless I have misunderstood how that works).

Reply:
Yes, it works that way.